usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/clients/GoogleCalendarManagerClient.java

```diff
@@ -0,0 +1,176 @@
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @return List containing all the Events from the specified Calendar
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public List<EventData> getAllEventsFromCalendar(String accessToken, String owner, String petName)
+        throws ExecutionException, InterruptedException {
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(GET);
+        StringBuilder response = taskManager.execute(BASE_URL + owner + "/" + petName, accessToken).get();
+        List<EventData> eventList = new ArrayList<>();
+        if (response.length() > 2) {
+            String jsonArray = response.substring(1, response.length() - 1);
+            String[] events = jsonArray.split(",\\{");
+            eventList.add(GSON.fromJson(events[0], EventData.class));
+            for (int i = 1; i < events.length; i++) {
+                events[i] = "{" + events[i];
+                eventList.add(GSON.fromJson(events[i], EventData.class));
+            }
+        }
+        return eventList;
+    }
+
+    /**
+     * Creates an Event in a specified Google Calendar.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @param eventData Event to create
+     * @return The response code
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    public int createEvent(String accessToken, String owner, String petName, EventData eventData)
+        throws ExecutionException, InterruptedException {
+        JSONObject reqJson = eventData.buildJson();
+        taskManager = taskManager.resetTaskManager();
+        taskManager.setTaskId(POST);
+        taskManager.setReqBody(reqJson);
+        StringBuilder response = taskManager.execute(BASE_URL + EVENT + owner + "/" + petName, accessToken).get();
+        return Integer.parseInt(response.toString());
+    }
+
+    /**
+     * Retrieves an Event in a specified Google Calendar.
+     * @param accessToken oauth2 token needed to access the Google Calendar
+     * @param owner Name of the owner of the pet
+     * @param petName Name of the pet the calendar belongs to
+     * @param eventId Id of the event to retrieve with key
+     * @return Event retrieved
+     * @throws ExecutionException When the retrieval fails
+     * @throws InterruptedException When the retrieval is interrupted
+     */
+    @Deprecated
+    public EventData getEvent(String accessToken, String owner, String petName, String eventId)
+        throws ExecutionException, InterruptedException {
+        Map<String, String> reqData = new HashMap<>();
+        reqData.put("eventId", eventId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "eventId" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5405224)